<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Result</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="graphImg">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="score">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>80</y>
      <width>180</width>
      <height>480</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>99%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>840</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Total Result</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>861</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButtonAL">
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Total</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonLA">
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Left Arm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRA">
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Right Arm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonLL">
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Left Leg</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRL">
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Right Leg</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonBO">
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Body</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>250</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Average</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="retry_btn">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>460</y>
      <width>135</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RETRY</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
